### ISSUE TITLE
No easy way to reach `Export` profile button

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#189342]
When exporting a profile, a SR user has to navigate the whole list to reach the `Export` button. We should either move the button to the top or have a keybinding/command which does this. 

cc @jooyoungseo

<img width="293" alt="Screenshot 2023-07-31 at 11 30 34 AM" src="https://github.com/microsoft/vscode/assets/29464607/1a4a40e3-f631-40b4-9d96-dea04ab27c02">

**Comment by jvesouza** [comment]
> When exporting a profile, a SR user has to navigate the whole list to reach the `Export` button. We should either move the button to the top or have a keybinding/command which does this.
> 
In my opinion a keybinding/command is good idea because I can activate from any place in the list.

**Comment by sandy081** [comment]
@meganrogge Is not the tab should take to Export button when the complete tree is focused instead of focusing the check box?

**Comment by meganrogge** [comment]
@sandy081 I'm not following

**Comment by meganrogge** [comment]
correct - pressing tab when my screen looks like the above causes focus to be lost rather than going to the button.

**Comment by sandy081** [comment]
Tab at this state is taking focus to the checkbox in the tree. IMO checkbox should be focused when the row is focused and you tab then.

@alexr00 Is it possible to change this behaviour?
